pkg/operator/bootstrapteardown/teardown.go

```diff
@@ -55,24 +59,35 @@ func TearDownBootstrap(config *rest.Config,
 		},
 	)
 
-	operatorSpec, _, _, err := operatorClient.GetOperatorState()
 	if err != nil {
-		return err
+		klog.Errorf("error in watching clusterversions: %#v", err)
```

Comment:
not sure how to handle this @hexfusion 

Do we want to return here because we errored?

Reply:
nevermind, this is changing in subsequent PRs